.github/workflows/labels-verify.yml

```
@@ -4,9 +4,14 @@ on:
   pull_request_target:
     types: [opened, labeled, unlabeled, synchronize]
 
+permissions:
+  contents: read 
```

lable-verify doesn't need the repo's content

```suggestion
  contents: none 
```